crates/ln-dlc-node/src/tests/just_in_time_channel/channel_close.rs

```diff
@@ -33,10 +33,12 @@ async fn ln_collab_close() {
     .await
     .unwrap();
 
+    let fee_sats = 10_000;
```

Comment:
🔧 Should this not come from the liquidity options or something like that?

Reply:
technically you are right, but on these tests we do not know about the liquidity options. This is only known on the application layer of the coordinator and the app. I am using a little hack to inject the fees in the liquidity request, but for that I have to do it manually.

see also https://github.com/get10101/10101/pull/1528/files#diff-a6f251cebbb64254ec28d79e916edc0275557625800fae9588e2cdc2598a545dR270-L276